spec/07-variables.md

```diff
@@ -225,6 +291,83 @@ global variable. If that alias is passed to the `unset` intrinsic
 ([§§](10-expressions.md#unset)), only that alias is destroyed. The next time that function
 is called, a new alias is created with the current value of the global
 variable.
+<a name="undefined-global-variables"></a>
+
+**Undefined Global Variables**
+A distinction is made based on the context where an undefined global variable is used. 
+
+***byVal Context***
+
+PHP does not implicitly define an undefined global variable and uses `NULL` as substitution value instead. Furthermore, an `E_NOTICE`, stating that the corresponding variable was undefined, is emitted except for the following cases: 
+
+1.   The variable is used as single expression
+       1.  in a statement.
+       2.  as argument of [isset](10-expressions.md#isset).
```

Comment:
This implies that no notice is issued if $a is used as single expression in isset(). This is true, but misleading - isset would generate no notices even if it weren't "single expression". Also, the term "single expression" is not clear.